en/power_module/sky-drones_smartap-pdb.md

```diff
@@ -0,0 +1,72 @@
+# Sky-Drones SmartAP PDB
+
+[SmartAP PDB](https://sky-drones.com/power/smartap-pdb.html) (Power Distribution Board) is used to simplify the distribution of power from one or more batteries to ESCs (motors), flight controller, and other peripherals.
+It also acts as a [power module](../power_module/README.md), enabling battery voltage and current measurements.
+SmartAP PDB makes connecting high-power lines easier and much more reliable.
+
+![SmartAP PDB](../../assets/hardware/power_module/sky-drones_smartap-pdb/smartap-pdb-top-side.jpg)
+
+## Specifications
+
+- Size: 65x65 mm, 4x M3 mounting holes
+- Input voltage up to 60 Volts (14S)
+- Capability to handle extremenly high currents (peak current up to 400A)
+- Power input from the main battery, possibility to connect up to 2 independent batteries
```

Comment:
How? You only have the two big pads. Or do you mean you can use top side and bottom side pads for two batteries, and these end up being in parallel?

Reply:
Yes, the idea is that power cables can be soldered to both top and bottom pads allowing to connect two batteries in parallel. We have customers who use even four batteries in parallel by connecting two to the top pads and two to the bottom pads.